scripts/install_docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+curl -sSL https://get.docker.com/ubuntu/ | sh
```

Comment:
Why not use sudo apt-get install docker.io (to avoid hardcoded path and pipe to shell)?